variables/shared.yaml

```diff
@@ -35,6 +35,8 @@ variables:
     value: "prod-$(Build.SourceBranchName)"
   - name: gitRepoName
     value: vh-${{ variables.git_repo_name }}
+  - name: stagingTagName
```

Comment:
This is in the staging yaml so not needed?

Reply:
removed duplicates, found one for Prod release as-well.